string ID
string CMD
Map sceneMap
Player[] players
---
Player[] players